src/net_processing.cpp

```diff
@@ -331,6 +331,9 @@ class PeerManagerImpl final : public PeerManager
     /** Send `addr` messages on a regular schedule. */
     void MaybeSendAddr(CNode& node, std::chrono::microseconds current_time);
 
+    /** Send `feefilter` message. */
+    void MaybeSendFeefilter(CNode& node, std::chrono::microseconds current_time) EXCLUSIVE_LOCKS_REQUIRED(cs_main);;
```

Comment:
minor nit: extra semi-colon at the end of method declaration

Reply:
fixed; thanks for catching that